### PR TITLE
[T] Tests for HDGalleryTiles

### PR DESCRIPTION
**Pull request by drapisarda**

### CODE REVIEW

File: tests/unit/components/HdGalleryTiles.spec.js [hunk 1]
```diff
@@ -8,12 +8,27 @@ describe('HdGalleryTiles', () => {
...
     expect(wrapper.html()).toMatchSnapshot();
   });
+
+  it('emits the proper event when the user clicks on an image', () => {
+    const chosenImageIndex = 2;
+    const image = wrapper.findAll('.gallery-tile').at(chosenImageIndex);
+    image.trigger('click');
+
+    expect(wrapper.emitted('itemClick')[0][0]).toBe(chosenImageIndex);
+  });
```
Comment (viniciuskneves) [inline tests/unit/components/HdGalleryTiles.spec.js]: I would add the same test covering this event as well: https://github.com/homeday-de/homeday-blocks/blob/develop/src/components/gallery/HdGalleryTiles.vue#L10

Comment (drapisarda) [reply tests/unit/components/HdGalleryTiles.spec.js]: Done!

File: tests/unit/components/HdGalleryTiles.spec.js [hunk 2]
```diff
@@ -8,12 +8,27 @@ describe('HdGalleryTiles', () => {
...
     });
   });
 
   it('renders component', () => {
     expect(wrapper.html()).toMatchSnapshot();
   });
+
+  it('emits the proper event when the user clicks on an image', () => {
+    const chosenImageIndex = 2;
+    const image = wrapper.findAll('.gallery-tile').at(chosenImageIndex);
```
Comment (ilyasmez) [inline tests/unit/components/HdGalleryTiles.spec.js]: Same as the other PR:
```js
const TITLE_SELECTOR = '.gallery-tile';
```

File: tests/unit/components/HdGalleryTiles.spec.js [hunk 3]
```diff
@@ -8,12 +8,27 @@ describe('HdGalleryTiles', () => {
...
+    image.trigger('click');
+
+    expect(wrapper.emitted('itemClick')[0][0]).toBe(chosenImageIndex);
+  });
+
+  it('emits the proper event clicking on the _more photos_ button', () => {
+    const button = wrapper.findAll('.gallery-tile__more-photos');
```
Comment (ilyasmez) [inline tests/unit/components/HdGalleryTiles.spec.js]: Same as the other PR:
```js
const MORE_PHOTOS_SELECTOR = '.gallery-tile__more-photos';
```

File: tests/unit/components/HdGalleryTiles.spec.js (hunk 1 above)
Comment (viniciuskneves) [reply tests/unit/components/HdGalleryTiles.spec.js]: I think you forgot to push it 😅

Comment (drapisarda) [reply tests/unit/components/HdGalleryTiles.spec.js]: FFFUUUUU

**Pull request merged**